### PR TITLE
Translation from Gimena/TEI

### PR DESCRIPTION
**Pull request by Gimena**

### CODE REVIEW

File: P5/Source/Specs/fileDesc.xml [hunk 1]
```diff
@@ -19,8 +19,7 @@ $Id$
   <desc versionDate="2007-05-02" xml:lang="zh-TW">包含對電子檔案的完整書目敘述。</desc>
   <desc versionDate="2008-04-05" xml:lang="ja">電子ファイルに関する完全な書誌情報を示す．</desc>
     <desc versionDate="2016-11-17" xml:lang="de">enthält die vollständige bibliografische Beschreibung einer elektronischen Datei.</desc>
-  <desc versionDate="2007-05-04" xml:lang="es">contiene una descripción bibliográfica completa del
-    archivo electrónico.</desc>
+  <desc versionDate="2021-10-15" xml:lang="es"/>
```
Comment (peterstadler) [inline P5/Source/Specs/fileDesc.xml:22]: is that removal on purpose?

Comment (hcayless) [reply P5/Source/Specs/fileDesc.xml:22]: That was probably an error. I'll revert it :-)
